README.md

```diff
@@ -1,233 +1,194 @@
 # WSO2 Micro Integrator Documentation
 
-This repository contains the source code for WSO2 Micro Integrator documentation.
-
-This repository is open and we welcome your contributions!
-
-To access the WSO2 Micro Integrator documentation site, visit https://mi.docs.wso2.com/.
+[![Documentation Site](https://img.shields.io/badge/Visit%20Docs-mi.docs.wso2.com-blue)](https://mi.docs.wso2.com/)
+[![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](LICENSE)
+
+This repository contains the source code for WSO2 Micro Integrator documentation. We welcome your contributions!
+
+## 📚 Table of Contents
+
+- [Getting Started](#getting-started)
+- [How to contribute](#contributing)
+  - [Quick Edits (Minor Changes)](#quick-edits-minor-changes)
+  - [Major Changes](#major-changes)
+- [Setting Up Your Development Environment](#setting-up-your-development-environment)
+  - [Prerequisites](#prerequisites)
+  - [Installation Steps](#installation-steps)
+- [Running the Project Locally](#running-the-project-locally)
+- [Troubleshooting](#troubleshooting)
+  - [MkDocs Command Issues](#mkdocs-command-not-found)
+  - [Image and Page Updates](#when-adding-new-images-or-pages)
+  - [Markdown Formatting](#formatting-help)
```

Comment:
Contributor License Agreement (CLA) is missing

Reply:
Added by e867ad1